src/main/webapp/script.js

```diff
@@ -233,25 +231,29 @@ function createRecipeElement(recipe) {
 function createRecipeCardAlerts(recipe) {
   const alertElements = document.createElement('div');
   alertElements.className = 'recipe-card-block';
-  
-  fetch('/profile').then(response => response.json()).then((message) => {
```

Comment:
I think we can figure out how to have nested fetches like this but obv we need to find a new approach.